images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -630,3 +630,56 @@ function Get-GitHubPackageDownloadUrl {
     return $downloadUrl
 }
 
+function Use-ChecksumСomparison {
+    param (
+        [Parameter(Mandatory=$true)]
+        [string]$localFileHash,
+        [Parameter(Mandatory=$true)]
+        [string]$distributorFileHash
+    )
+
+    Write-Verbose "Performing checksum verification"
+
+    if ($localFileHash -ne $distributorFileHash) {
+        Write-Host "hash must be equal to: ${distributorFileHash}"
+        Write-Host "actual hash is: ${localFileHash}"
+        throw 'Checksum verification failed, please rerun install'
+    }else{
+        Write-Host "Checksum verification passed"
+    }
+}
+
+function Get-HashFromGitHubReleaseBody {
+    param (
+        [string]$RepoOwner,
+        [string]$RepoName,
+        [Parameter(Mandatory=$true)]
+        [string]$FileName,
+        [string]$Url,
+        [string]$Version = "latest",
+        [boolean]$IsPrerelease = $false,
+        [int]$SearchInCount = 100,
+        [string]$Delimiter = ' | ',
+        [int]$WordNumber = 1
+    )
+
+    if ($Url) {
+        $releaseUrl = $Url
+    }else{
+        if ($Version -eq "latest") {
+            $releaseUrl = "https://api.github.com/repos/${RepoOwner}/${RepoName}/releases/latest"
+        }else{
+            $json = Invoke-RestMethod -Uri "https://api.github.com/repos/${RepoOwner}/${RepoName}/releases?per_page=${SearchInCount}"
```

Comment:
Will it be possible to utilize `Get-GitHubPackageDownloadUrl` helper function here?

Reply:
Nope. Different targets.